src/aat/resources/features/F-042/S-249.td.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Trigger the aboutToStart callback event for a caseworker for an existing case which has not started yet.",
+  "_guid_": "S-249",
+  "_extends_": "F-042_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is prepared with an invalid START_EVENT",
+    "has the 422 return code"
+  ],
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]",
+      "cid": "[[DYNAMIC]]",
+      "jid": "BEFTA_JURISDICTION_1",
+      "ctid": "BEFTA_CASETYPE_1_1",
+      "etid": "CREATE"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_422_Response",
+    "headers": {
+      "Content-Length": "[[ANY]]",
+      "Content-Encoding": "gzip",
+      "Content-Type": "application/json;charset=UTF-8"
+    },
+    "body": {
+      "exception": "uk.gov.hmcts.ccd.endpoint.exceptions.ValidationException",
+      "timestamp": "[[ANY]]",
+      "status": 422,
+      "error": "Unprocessable Entity",
+      "message": "The case status did not qualify for the event",
+      "path": "[[ANY]]",
+      "details": null,
+      "callbackErrors": null,
+      "callbackWarnings": null
```

Comment:
status, error, callbackErrors and callbackWarnings are included in Common_422_Response_Body, can be removed from here

Reply:
timestamp and path are included in Common_4XX_Response_Body, can also be removed from here

Reply:
Removed the common fields from scenario file.